core/src/replay_stage.rs

```diff
@@ -2005,7 +2013,7 @@ impl ReplayStage {
         banking_tracer: &Arc<BankingTracer>,
         has_new_vote_been_rooted: bool,
         track_transaction_indexes: bool,
-    ) {
+    ) -> bool {
```

Comment:
Is this just for unit test validation?

Reply:
yep, just makes it easier to unit test. Also seems more in line with the name: MAYBE_start_leader should tell us if it started the leader bank.